src/Dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -1,48 +1,55 @@
+import { useEffect, useState } from "react";
 import Header from "../Header/Header";
+import axios from "axios";
+import { useUser } from "../../../context/AuthContext";
+import DonutChart from "react-donut-chart";
 
 export default function Dasxboard() {
+  const { Token } = useUser();
+  const [todoCount, settodoCount] = useState<number>(0);
+  const [progressCount, setprogressCount] = useState<number>(0);
+  const [doneCount, setdoneCount] = useState<number>(0);
+  const [activatedEmployeeCount, setactivatedEmployeeCount] = useState<number>(0);
+  const [deactivatedEmployeeCount, setdeactivatedEmployeeCount] = useState<number>(0);
+
+  const getTasksCount = async () => {
+    try {
+      let response = await axios.get(
```

Comment:
It is a good practice to always have const variables,  unless we are sure that we are going to change a variable latter.
In this case, we will not change `response` , so it should be a constant.